hivemq/assets/dashboards/hivemq.json

```diff
@@ -0,0 +1,682 @@
+    "widgets": [
+        {
+            "height": 6,
+            "sizing": "zoom",
+            "type": "image",
+            "url": "/static/images/saas_logos/bot/hivemq@2x.png",
+            "width": 21,
+            "x": 1,
+            "y": 1
+        },
+        {
+            "check": "hivemq.can_connect",
+            "group_by": [],
+            "grouping": "cluster",
+            "height": 10,
+            "tags": [
+                "*"
+            ],
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Connected Nodes",
+            "type": "check_status",
+            "width": 21,
+            "x": 1,
+            "y": 8
+        },
+        {
+            "height": 8,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 0,
+                "requests": [
+                    {
+                        "aggregator": "last",
+                        "q": "sum:hivemq.networking.connections.current{*}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Current Connections",
+            "type": "query_value",
+            "width": 21,
+            "x": 1,
+            "y": 19
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "custom_unit": "/s",
+                "precision": 0,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "sum:hivemq.messages.incoming.total.count{*}.as_rate()"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Incoming Message Rate",
+            "type": "query_value",
+            "width": 21,
+            "x": 1,
+            "y": 41
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "custom_unit": "/s",
+                "precision": 0,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "sum:hivemq.messages.outgoing.total.count{*}.as_rate()"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Outgoing Message Rate",
+            "type": "query_value",
+            "width": 21,
+            "x": 1,
+            "y": 54
+        },
+        {
+            "bgcolor": "vivid_yellow",
```

Comment:
Should the category headers be gray? https://datadoghq.dev/integrations-core/guidelines/dashboards/#metric-categories

Yellow is typically reserved for notes/memos/comments.